docs/configuration.md

```diff
@@ -422,7 +433,7 @@ Control the verbosity of logs.
 logging:
   level: error
 ```
-There exists a log level hierarchy in order as `trace`, `debug`, `info`, `warn`, `error`, `fatal`, and `notify`. When the level is set to `trace` logs will be created for all possible levels. Whereas if the level is set to `fatal`, the only entries logged will be `fatal` and `notify`. The default value is `error`.
+There exists a log level hierarchy in order as `trace`, `debug`, `info`, `warn`, and `error`. When the level is set to `trace` logs will be created for all possible levels. Whereas if the level is set to `warn`, the only entries logged will be `warn` and `error`. The default value is `error`.
```

Comment:
What was your thinking for changing this paragraph? We still have fatal and notify logs in hdb. This is not the same for clustering log level - there aren't as many log levels for that.

Reply:
oh I was looking in the natsTerms.js, instead of hdbTerms.js. my fault. will change back